src/Cli/dotnet/commands/dotnet-new/PostActions/DotnetAddJsonPropertyPostActionProcessor.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Abstractions.PhysicalFileSystem;
+using Microsoft.TemplateEngine.Cli.PostActionProcessors;
+
+namespace Microsoft.DotNet.Tools.New.PostActionProcessors
+{
+    internal class DotnetAddJsonPropertyPostActionProcessor : PostActionProcessorBase
```

Comment:
imo, this post action will be better in https://github.com/dotnet/sdk/tree/main/src/Cli/Microsoft.TemplateEngine.Cli/PostActionProcessors as it has no dependency to `dotnet`.

Reply:
I can do that.  Do I need to move the tests to another location as well ?

Reply:
yes, please

Reply:
Any suggestion on where I could put those unit-tests ?
I've moved the `DotnetAddJsonPropertyPostActionProcessor` from `dotnet/commands/dotnet-new`  to `Microsoft.TemplateEngine.Cli/PostActionProcessors` .  There are already other post-action processors there, so I figured I'll put the unittests in the same location as the tests of the other postaction processors, but it looks like the other postactions (like ChmodPostActionProcessor) does not have any tests ? :?

Reply:
(There are some build errors right now because I need to move the unit-tests :) )

Reply:
It seems like the other post-process do not have tests. I have opened a issue about it so we can add them later (https://github.com/dotnet/templating/issues/6004). For now, you can add them to `sdk\src\Tests\Microsoft.TemplateEngine.Cli.UnitTests` .

Reply:
I have been corrected, the tests for those post actions are in `src/Tests/dotnet-new.Tests/PostActionTests.cs`, so you can add tests there.

Reply:
These are integration-tests, but not unit-tests.
I do have integration-tests and unit-tests for this post-action, so for the moment, I'll put the unit-tests in `Microsoft.TemplateEngine.Cli.UnitTests`.

Reply:
That will be correct place for unit tests